app/lib/emoji.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'singleton'
+
+class Emoji
+  include Singleton
+
+  def initialize
+    data = Oj.load(File.open(File.join(Rails.root, 'node_modules', 'emojione', 'emoji.json')))
```

Comment:
I'm not sure what else we can do. There is a problem with this, of course - now you must install node dependencies even on machines that would only run ruby workers.